ngrinder-controller/build.gradle

```diff
@@ -108,6 +108,12 @@ task convert_cr_lf {
     }
 }
 
+processResources {
+    filesMatching("ngrinder-sh/agent/run_agent_internal.*") {
+        expand(["ngrinder_core": String.format("lib/ngrinder-core-%s.jar", project.version)])
+    }
+}
```

Comment:
Some grinder's class is customized for ngrinder.
so, agent must load ngrinde-core before grinder's